cmd/manager/main.go

```diff
@@ -75,6 +75,9 @@ func main() {
 	var logAPIRequests bool
 	pflag.BoolVar(&logAPIRequests, "logAPIRequests", false, "Log API requests performed by the operator.")
 
+	var runLocalBuild bool
+	pflag.BoolVar(&runLocalBuild, "runLocalBuild", false, "Run the operator locally.")
```

Comment:
Instead of calling the flag `runLocalBuild`, I suggest using `leader-elect`, since this is a pretty standard flag used across different controller managers (including the core [kubernetes controller manager](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/)).